src/main/java/lotto/model/LottoSize.java

```diff
@@ -0,0 +1,32 @@
+package lotto.model;
+
+public class LottoSize {
+    private final int size;
+
+    public LottoSize(int size) {
+        validateLottoSize(size);
+        this.size = size;
+    }
+
+    public int getRestOfLottoSize(int availableSize) {
+        int restSize = availableSize - size;
+        validateLottoSize(restSize);
+        return restSize;
+    }
+
+    private void validateLottoSize(int size) {
+        if (size < 0) {
+            throw new IllegalArgumentException("로또는 0장 이상 구매하셔야 합니다.");
+        }
+    }
+
+    public void validateLottoSizeWithMoney(Money money) {
+        if (money.getBuyableLottoSize() < size) {
+            throw new IllegalArgumentException("수동 로또의 수가 구입 금액을 넘을 수 없습니다.");
```

Comment:
`LottoSize` 객체에서 수동과 자동의 개념을 지우셨지만 예외 메시지에는 하나 남아있네요!!

꼭 수동 로또가 아니라 로또의 수에 대한 메시지로 고쳐도 될거 같아요. 

```suggestion
            throw new IllegalArgumentException("로또의 수가 구입 금액을 넘을 수 없습니다.");
```